api/representation.go

```diff
@@ -195,6 +195,13 @@ func (tr TaskRequest) ToTaskConfig() (config.TaskConfig, error) {
 
 	// Enterprise
 	tc.TFVersion = tr.Task.TerraformVersion
+	if tr.Task.TerraformCloudWorkspace != nil {
```

Comment:
So I'm sure this is true, but just double checking

Since this is OSS, and since it gets converted to a config, we will catch the "error of an enterprise config" as part of the config validation right?

Reply:
Yup, validation will be handled later by the server and will look like this:
```
{
    "error": {
        "message": "unsupported configuration 'terraform_cloud_workspace' for task \"mkam_test_1\". This option is available for Consul-Terraform-Sync Enterprise when using the Terraform Cloud driver"
    },
    "request_id": "8d1e47f1-d400-ef35-326c-84ebe93bf487"
}
```

Reply:
That message will only be outputted if we configure it in [processUnusedConfigKeys](https://github.com/hashicorp/consul-terraform-sync/blob/main/config/decode.go#L14-L40) right?